### PR TITLE
[stable/kubernetes-dashboard] Allow the use of a specific image digest

### PR DESCRIPTION
**Pull request by drastawi**
What this PR does / why we need it:

Using a digest ensures the image being pulled has not been compromised and allows testing release against images without having to tag them.

Special notes for your reviewer:

This pr is closely based on #8910

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] title of the PR contains starts with chart name e.g. `[stable/chart]`

### CODE REVIEW

File: stable/kubernetes-dashboard/README.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ The following table lists the configurable parameters of the kubernetes-dashboar
 |-------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------|
 | `image.repository`                  | Repository for container image                                                                                              | `k8s.gcr.io/kubernetes-dashboard-amd64`                                    |
 | `image.tag`                         | Image tag                                                                                                                   | `v1.10.1`                                                                  |
+| `image.digest`                      | Image digest                                                                                                                | `{}`                                                                       |
```
Comment (kfox1111) [inline stable/kubernetes-dashboard/README.md]: {} means array. Its a blank string, not an empty array.

**Comment by kfox1111** [comment]
Has one minor issue but looks good in general. Thanks for contributing.

File: stable/kubernetes-dashboard/README.md (hunk 1 above)
Comment (davidkarlsen) [inline stable/kubernetes-dashboard/README.md]:
```suggestion
| `image.digest`                      | Image digest                                                                                                                | ``                                                                       |
```

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Comment by jlegrone** [comment]
Placing on hold until https://github.com/helm/charts/issues/13449 is resolved.

/hold

**Pull request closed without merging**